packages/hdwallet-native/src/ethereum.ts

```diff
@@ -43,8 +44,8 @@ export function MixinNativeETHWallet<TBase extends core.Constructor>(Base: TBase
 
     ethWallet: Wallet;
 
-    ethInitializeWallet(mnemonic: string): void {
-      this.ethWallet = Wallet.fromMnemonic(mnemonic);
+    ethInitializeWallet(seed: string): void {
+      this.ethWallet = new Wallet(HDNode.fromSeed(seed).derivePath(defaultPath));
```

Comment:
So fromMnemonic is just super slow for some reason?

Reply:
yep